.github/workflows/CD.yml

```diff
@@ -61,10 +65,69 @@ jobs:
           CONTAINER_REGISTRY_PASSWORD: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
           CONTAINER_REGISTRY_USER: ${{ secrets.CONTAINER_REGISTRY_USER }}
         run: docker login quay.io --username $CONTAINER_REGISTRY_USER --password $CONTAINER_REGISTRY_PASSWORD
-      - name: Build and Publish JVM Container
+      - name: Build and Publish JVM Container - All in One
         shell: bash
         run: |
-          docker build -f runner/src/main/docker/Dockerfile.jvm -t $OUTPUT_CONTAINER_NAME-jvm runner/
-          docker push $OUTPUT_CONTAINER_NAME-jvm
-          docker image rm -f $OUTPUT_CONTAINER_NAME-jvm
-#
+          echo "Building and pushing docker image for All-in-One"
+          docker build -f docker/Dockerfile.jvm -t $ALL_IN_ONE_CONTAINER_NAME-jvm runner/
+          docker push $ALL_IN_ONE_CONTAINER_NAME-jvm
+          docker image rm -f $ALL_IN_ONE_CONTAINER_NAME-jvm
```

Comment:
@r00ta This `image rm -rf` means that the following steps to build a component have to do the entire `docker build` again. If we remove it, it means the following components should benefit from the Docker build cache and make the whole pipeline much quicker.

Reply:
Yeah it's a fair point. 
I've removed the ` docker image rm -f` from the steps and I've added a last clean up step. Wdyt @robpblake ?

Reply:
@r00ta Just thinking about it..do we even need to do the clean-up of the container image at all? I assume the GH Action runner filesystem is "fresh" for each build?